src/components/Header/index.js

```diff
@@ -210,7 +203,6 @@ const GordonHeader = ({ onDrawerToggle }) => {
               />
               {requiresAuthTab('People', <PeopleIcon />)}
               {/* {requiresAuthTab('Timesheets', WorkIcon)} */}
```

Comment:
Should we put Timesheets back in the header now that we're removing wellness?

Reply:
I didn't know that was why it was removed. Yes! Let's add it back.
```suggestion
              {requiresAuthTab('Timesheets', WorkIcon)}
```